cloneall

```diff
@@ -6,7 +6,16 @@
 #
 . ./include/bash/common.sh
 
-
+# return gitlab url bases on provided authentication or not
```

Comment:
Hi Nguyen,
gitlab and soco should not be part of the OSS repository.

Maybe a solution is a conditional import which loads this functionality only when the gitlab build respository is available.

Thank you,
 Thomas

Reply:
Hi Thomas,
This is solved in the commit 7b3d086f7cb26a80db1e078038ccad69dfeea19f